### PR TITLE
[#168316347] Added confirm alert pagopa test

### PR DESCRIPTION
**Pull request by matgentili**
Added confirm alert pagopa test

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#168316347](https://www.pivotaltracker.com/story/show/168316347): Come CIT, quando abilito l'ambiente di test di pagopa, voglio avere un confirm prima del popup

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by matgentili** [comment]
BEFORE:
pagopa before

AFTER ITA:

pagopa after ita

AFTER ENG:

pagopa after eng

File: ts/screens/profile/ProfileMainScreen.tsx [hunk 1]
```diff
@@ -177,11 +177,28 @@ class ProfileMainScreen extends React.PureComponent<Props> {
   };
 
   private onPagoPAEnvironmentToggle = (enabled: boolean) => {
-    this.props.setPagoPATestEnabled(enabled);
-    this.props.showModal(
-      <AlertModal
-        message={I18n.t("profile.main.pagoPaEnvironment.alertMessage")}
-      />
+    Alert.alert(
```
Comment (Undermaken) [inline ts/screens/profile/ProfileMainScreen.tsx]: We should show the warning only when user tries to enable this feature.
As it is it always shows the warning, also when the user disables it.

File: locales/it/index.yml [hunk 2]
```diff
@@ -144,13 +144,15 @@ profile:
       confirmMessage: Le funzionalità sperimentali potrebbero rendere l'applicazione instabile. Sei sicuro di volerle abilitare?
       bannerMessage: Stai usando funzioni sperimentali che potrebbero essere incomplete o instabili.
     pagoPaEnvironment:
+      pagoPaEnv: Ambiente PagoPA di Test
+      pagoPAEnvAlert: La modifica richiede il reboot dell'app
```
[the remark this answers is not in the document]
Comment (matgentili) [reply locales/it/index.yml]: Ok ;)

File: ts/screens/profile/ProfileMainScreen.tsx (hunk 1 above)
Comment (matgentili) [reply ts/screens/profile/ProfileMainScreen.tsx]: Sorry, I didn't think about it. Fixed :)

File: ts/screens/profile/ProfileMainScreen.tsx [hunk 3]
```diff
@@ -177,12 +177,35 @@ class ProfileMainScreen extends React.PureComponent<Props> {
...
+                  )}
+                />
+              );
+            }
+          }
+        ],
+        { cancelable: false }
+      );
+    } else {
+      this.props.setPagoPATestEnabled(enabled);
```
Comment (Undermaken) [inline ts/screens/profile/ProfileMainScreen.tsx]: You should always show that a reboot is needed 

switch changes
-  if ON -> show warning
    - if CONFIRM warning -> show reboot is needed
- if OFF -> show reboot is needed

Comment (matgentili) [reply ts/screens/profile/ProfileMainScreen.tsx]: Ok, done

**Pull request merged**